diff --git a/Makefile.am b/Makefile.am
index d1fbf57..172f848 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -98,11 +98,11 @@ BitVectorTest_LDADD = libcommon.la $(SQLITE_LA)
 
 InterthreadTest_SOURCES = InterthreadTest.cpp
 InterthreadTest_LDADD = libcommon.la $(SQLITE_LA)
-InterthreadTest_LDFLAGS = -lpthread -lcoredumper 
+InterthreadTest_LDFLAGS = -lpthread  
 
 SocketsTest_SOURCES = SocketsTest.cpp
 SocketsTest_LDADD = libcommon.la $(SQLITE_LA)
-SocketsTest_LDFLAGS = -lpthread -lcoredumper 
+SocketsTest_LDFLAGS = -lpthread  
 
 TimevalTest_SOURCES = TimevalTest.cpp
 TimevalTest_LDADD = libcommon.la
@@ -117,10 +117,10 @@ ConfigurationTest_SOURCES = ConfigurationTest.cpp
 ConfigurationTest_LDADD = libcommon.la 	$(SQLITE_LA)
 
 SelfDetectTest_SOURCES = SelfDetectTest.cpp
-SelfDetectTest_LDADD = libcommon.la $(SQLITE_LA) -lcoredumper 
+SelfDetectTest_LDADD = libcommon.la $(SQLITE_LA)  
 
 UnixSignalTest_SOURCES = UnixSignalTest.cpp
-UnixSignalTest_LDADD = libcommon.la $(SQLITE_LA) -lcoredumper 
+UnixSignalTest_LDADD = libcommon.la $(SQLITE_LA)  
 
 # ReportingTest_SOURCES = ReportingTest.cpp
 # ReportingTest_LDADD = libcommon.la $(SQLITE_LA)
diff --git a/UnixSignal.cpp b/UnixSignal.cpp
index de55b3c..63ec31d 100644
--- a/UnixSignal.cpp
+++ b/UnixSignal.cpp
@@ -30,7 +30,6 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
-#include <google/coredumper.h>
 
 #include "UnixSignal.h"
 #include "Logger.h"
@@ -65,7 +64,6 @@ void UnixSignal::Handler(int sig)
 		snprintf(buf, sizeof(buf)-1, "%s.%d", mCoreFile.c_str(), getpid());
 	    else
 		snprintf(buf, sizeof(buf)-1, "%s", mCoreFile.c_str());
-	    WriteCoreDump(buf);
 
 	    // and save the files if needed
 	    if (mSaveFiles)
